test/kubelessInfo.test.js

```diff
@@ -139,31 +139,6 @@ describe('KubelessInfo', () => {
         .get(`/apis/kubeless.io/v1beta1/namespaces/${f.namespace}/functions/${f.id}`)
         .reply(200, _.find(allFunctions, (ff) => ff.metadata.name === f.id));
     });
-
-
-    // Mock call to get.ingress
-    _.each(functions, f => {
```

Comment:
I believe this the cause of the issue you are facing with the test. This is just a mock that returns the ingress if requested, why did you remove it?

Reply:
I removed it because the ingress is now created by kubeless ... so I think the serverless-kubeless plugin is not more responsible for the ingress creation. Right?

Reply:
I mean, the kubeless plugin is unaware of the ingress created by kubeless.

Reply:
yeah, well, the result at the end of the day is the same. An Ingress with the name of the function is created so that's what this mock is doing.

If you want to remove this, you need to create the mock to return the HTTP trigger instead and change the `getInfo` code to request the HTTP trigger instead. I think that is more complicated to do so, that's why I suggest to just leave this as it was but if you want to run the extra mile and adapt that code, that's also fine ;)

Reply:
Ok ;)
We'll catch later.

Reply:
“Later” is the new “next week” 😏 Sorry For the delay @andresmgot. I will work on this pr next monday.